### ISSUE TITLE
question (unsure of something)

### ISSUE THREAD

**Issue by MarioMasta64** [YANBForwarder/YANBF#37]
where is the default path for the rom files? i cant seem to figure that out

is it sdcard root? is it a folder named roms? how do i format the filepaths in the gui if im setting it myself? im a little lost.

**Comment by MarioMasta64** [comment]
i figured it out how to set paths but there is no way to know what youre supposed to set it to and no default path mentioned so i would leave this open simply for that note
/roms/filename.nds

**Comment by lifehackerhansol** [comment]
The default path is the current location of your ROM. Ideally, you would drag and drop the ROM that is already on your SD card into the application.

For people who use FTP or similar to manage their files, since the SD card is not plugged into the PC, this SD card drag-n-drop does not work. So I allowed a way for these people to specify their own ROM path, which is equivalent to the location on your SD card. 

So to summarize:
- If your SD card is plugged into your PC, drag and drop the ROM from the SD card to the PC. It'll handle it automatically.
- If you use a wireless method to transfer your files, manually set the ROM path to where your ROM is on the SD card.

I hope this helps, and sorry about the late response!
